module/signature/checksum.go

```diff
@@ -0,0 +1,74 @@
+package signature
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// CheckSumLen is fixed to be 16 bytes, which is also md5.Size
+const CheckSumLen = 32
```

Comment:
If we use CRC32, the length would be 4. 

```suggestion
// CheckSumLen is fixed to be 4 bytes
const CheckSumLen = 4
```